Extensions/3D/Model3DRuntimeObject3DRenderer.ts

```diff
@@ -347,6 +348,27 @@ namespace gdjs {
       this._action.play();
       // Make sure the first frame is displayed.
       this._animationMixer.update(0);
+      if (wasPaused) {
+        this.pauseAnimation();
```

Comment:
Shouldn't this be:
```suggestion
        this.resumeAnimation();
```
?

Reply:
It keeps the "pause" state untouched because `play` is automatically starting the animation.

Reply:
Worth a comment